e2e/mock-api/project.json

```diff
@@ -8,9 +8,10 @@
       "outputs": ["{options.outputPath}"],
       "options": {
         "outputPath": "dist/e2e/mock-api",
-        "main": "e2e/mock-api/src/index.mjs",
+        "main": "e2e/mock-api/src/index",
```

Comment:
Do we have any references to the minimal Node.js version that corresponds to this change?